ios/Sources/Model/FeedContents.swift

```diff
@@ -0,0 +1,17 @@
+import DroidKaigiMPP
+
+public struct FeedContents: Equatable {
+    public struct Content: Equatable {
+        public var item: FeedItemType
+        public var isFavorited: Bool
+
+        public init(item: FeedItemType, isFavorited: Bool) {
+            self.item = item
+            self.isFavorited = isFavorited
+        }
+    }
+
+    public var contents: [Content]
+    public var favorites: Set<String>
```

Comment:
Just in case, this is fine with this, right?

Reply:
Basically, this PR's definitions are based on KMM module.
So I think this is fine.